syscall/unistd.c

```diff
@@ -265,7 +276,13 @@ int oe_close(int fd)
         OE_RAISE_ERRNO(oe_errno);
 
     if ((ret = desc->ops.fd.close(desc)) == 0)
+    {
+        // Notify epoll instances that this fd has been closed.
+        oe_fdtable_foreach(
```

Comment:
Does it matter whether the notification happens after the close or whether it happens immediately prior to close?

Reply:
I'm not sure. I've put it after the close because the close could fail.